### PR TITLE
[Blog] Add 2.8.1 announcement blog

### PR DESCRIPTION
**Pull request by Anonymitaet**

### CODE REVIEW

**Review by hangc0276 — APPROVED** [review]

File: site2/website/blog/2021-09-23-Apache-Pulsar-2-8-1.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+# Notable bug fixes and enhancements
+
+## Broker
+
+### Precise publish rate limit takes effect as expected. [PR-11446](https://github.com/apache/pulsar/pull/11446)
+
+**Issue**: Previously, when setting precise publish rate limits on topics, it did not work.
+
+**Resolution**: Implemented a new `RateLimiter` using the `LeakingBucke` and `FixedWindow` algorithms.
```
Comment (eolivelli) [inline site2/website/blog/2021-09-23-Apache-Pulsar-2-8-1.md:28]: `LeakingBucke`, is this a typo. ?

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: site2/website/blog/2021-09-23-Apache-Pulsar-2-8-1.md (hunk 1 above)
Comment (Anonymitaet) [reply site2/website/blog/2021-09-23-Apache-Pulsar-2-8-1.md]: Good catch! I've updated it to `LeakingBucket`. Thanks!

Comment (Anonymitaet) [reply site2/website/blog/2021-09-23-Apache-Pulsar-2-8-1.md]: @eolivelli do you have more suggestions on this PR?

Comment (Anonymitaet) [reply site2/website/blog/2021-09-23-Apache-Pulsar-2-8-1.md]: @eolivelli if no more comments, I'll merge this PR.

**Review by eolivelli — APPROVED** [review]

**Pull request merged**